cypress/locators/menu-list/locators.js

```diff
@@ -1,6 +1,7 @@
 // component preview locators
 export const MENU_LIST_CHILDREN = 'div[data-component="menu-list"][class="carbon-menu-list"]';
-export const MENU_TITLE = '#story-root > div[data-component="menu-list"] > [data-component="link"]';
+export const MENU_TITLE = '[data-element="title"]';
```

Comment:
I think we already have one locator in `common` locators

Reply:
I've refactored it, instead of `'[data-element="title"]'` I've used `getDataElementByValue`